subsys/bluetooth/mesh/shell.c

```diff
@@ -794,15 +794,56 @@ static void print_unprovisioned_beacon(uint8_t uuid[16],
 			(uri_hash == NULL ? 0 : *uri_hash));
 }
 
+#if defined(CONFIG_BT_MESH_PB_GATT_CLIENT)
+static void pb_gatt_unprovisioned(uint8_t uuid[16],
+				      bt_mesh_prov_oob_info_t oob_info)
+{
+	char uuid_hex_str[32 + 1];
+
+	bin2hex(uuid, 16, uuid_hex_str, sizeof(uuid_hex_str));
+
+	shell_print_ctx("UUID %s, OOB Info 0x%04x", uuid_hex_str, oob_info);
+}
+
+static int cmd_provision_pb_gatt(const struct shell *sh, size_t argc,
+				 char *argv[])
+{
+	static uint8_t uuid[16];
+	uint8_t attention_duration;
+	uint16_t net_idx;
+	uint16_t addr;
+	size_t len;
+	int err;
+
+	len = hex2bin(argv[1], strlen(argv[1]), uuid, sizeof(uuid));
```

Comment:
I guess it is worth adding length checking. This is kind of open door for hackers :)  It is possible to play with uuid value to get the necessary `len` to set in zero some keys etc

Reply:
Or just corrupt memory in device remotely with unpredictable consequences.

Reply:
I see that it already exists in other places. It's best to solve this problem in a separate pr.